pkg/build/api/types.go

```diff
@@ -10,8 +10,12 @@ import (
 const (
 	// BuildAnnotation is an annotation that identifies a Pod as being for a Build
 	BuildAnnotation = "openshift.io/build.name"
+	// DeprecatedBuildLabel is old value of BuildLabel, it'll be removed in OpenShift 3.1.
+	DeprecatedBuildLabel = "build"
 	// BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
-	BuildLabel = "build"
+	BuildLabel = "io.openshift.build.name"
```

Comment:
Deployments also have a label used on pods related to a deployment: `openshift.io/deployer-pod-for.name`